src/common/components/editor-toolbar/index.tsx

```diff
@@ -256,11 +261,11 @@ export class EditorToolbar extends Component<Props> {
   };
 
   bold = () => {
-    this.insertText("**", "**");
+    this.insertText("*", "*");
```

Comment:
Why this is modified? @Adesojisouljay

Reply:
Ah, I don't know how i=I broke this. fixed